README.md

```diff
@@ -76,6 +80,24 @@ docker-compose run --rm  -v $(pwd)/rdf:/fuseki/rdf/ --entrypoint="sh /fuseki/scr
 # Start the containers again
 ```
 
+### Trigger intent via endpoint
+
+With all servers up and running, all intents can be triggered via the ```trigger_intent``` endpoint of your Rasa
+conversation. For instance, we include an intent for retrieving the 'MOTD' (Mot-of-the-day) greeting, which are 
+stored in the [motd.yml](data/motd/motd.yml) file. Different bot greetings are produced for different given dates and 
+languages. 
+
```

Comment:
MOTD: Message-of-the-day

Reply:
Haha, Daniel calls it Mot of the day. I'll go with whatever you guys agree on.

Reply:
It could be "match of the day", but we will go with "message of the day":)